dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala

```diff
@@ -7,39 +7,37 @@ trait LogicalFunctionTokenizer {
 
   def tokenizeLogicalFunction(col: LogicalFunction)(implicit ctx: TokenizeContext): String =
     (col.left.asOption, col.right.asOption) match {
-      case (None, None)        => "1"
-      case (Some(left), None)  => tokenizeColumn(left)
-      case (None, Some(right)) => tokenizeColumn(right)
+      case (None, None)       => "1"
+      case (Some(left), None) => tokenizeColumn(left)
+      case (None, Some(right)) =>
+        if (right.isConstTrue) tokenizeColumn(right)
+        else tokenize(right, col.operator)
       case (Some(left), Some(right)) =>
         col.operator match {
           case And =>
-            if (left.isConstTrue)
-              tokenizeColumn(right)
-            else if (right.isConstTrue)
-              tokenizeColumn(left)
+            if (left.isConstTrue) tokenizeColumn(right)
```

Comment:
Use `tokenize`?

Reply:
Well, the clause is a constant, so why putting (possible) brackets around it?